### DISCUSSION TITLE
Save filters

### DISCUSSION THREAD

**Discussion by Voldemorten** [directus/directus#12278] [Archive]
It would be very convenient, if you could save a filter in the app. 

![image](https://user-images.githubusercontent.com/8776117/159280057-9e789891-dd44-4929-a921-0024f8d62ec5.png)

Above is a UI suggestion of it, where filters are divided into "shared" (available for all users) and personal (available only for me). And then when you press one of these filters, the view is updated with the filter applied.

**Comment by benhaynes** [comment]
Have you seen our Bookmarks & Presets feature? It does exactly that. 🙂 

https://docs.directus.io/app/content-collections/#bookmarking-item-presets

**Reply by Voldemorten** [reply]
How could i miss it. That’s exactly what I needed! Thanks Ben!

**Reply by Vander-Ng** [reply]
Bookmarking is great, but your idea of sharing it with other users is even greater, can you do sharing with the current bookmark service?  I couldn't find a way to share with others....

**Reply by Vander-Ng** [reply]
Oh, just found that it could be done by setting the scope in Bookmark & Presets, nice feature!
<img width="1087" alt="Screen Shot 2022-04-09 at 6 17 16 PM" src="https://user-images.githubusercontent.com/78061132/162567772-471366d9-a2a5-47f3-9708-c9f2e2a7f067.png">
